tests/cmake-consumer/consumer.cpp

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
+
+// Check that OCIO library can be consumed by a cmake build
+
+#include <cassert>
+#include <iostream>
```

Comment:
[Minor] Are these two includes needed?

Reply:
Will remove `<iostream>` good catch, I seem to remember `cassert` is needed at least on Windows.